src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeConstructorInfo.CoreCLR.cs

```diff
@@ -118,6 +119,15 @@ public override string ToString()
 
             return m_toString;
         }
+
+        public override bool Equals(object? obj) =>
+            object.ReferenceEquals(this, obj) ||
```

Comment:
```suggestion
            ReferenceEquals(this, obj) ||
```
`object.` is unnecessary